packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2931,7 +2936,35 @@ export default class Meeting extends StatelessWebexPlugin {
           success: `${LOG_HEADER} Successfully send roap media request`,
           failure: `${LOG_HEADER} Error joining the call on send roap media request, `
         }))
+        .then(() => {
+          const {peerConnection} = this.mediaProperties;
+
+          return new Promise((resolve, reject) => {
+            if (peerConnection.iceConnectionState === ICE_STATE.CONNECTED) {
+              LoggerProxy.logger.info(`${LOG_HEADER} PeerConnection CONNECTED`);
 
+              resolve(peerConnection);
+
+              return;
+            }
+            // Check if Peer Connection is STABLE (connected)
+            const stabilityTimeout = setTimeout(() => {
+              if (peerConnection.iceConnectionState !== ICE_STATE.CONNECTED) {
+                reject(new Error(`Unable to add media. PeerConnection NOT CONNECTED after ${PC_BAIL_TIMEOUT / 1000} seconds.`));
+              }
+              else {
+                LoggerProxy.logger.info(`${LOG_HEADER} PeerConnection CONNECTED`);
+                resolve(peerConnection);
+              }
+            }, PC_BAIL_TIMEOUT);
+
+            this.on(EVENT_TRIGGERS.MEETING_MEDIA_CONNECTED, () => {
```

Comment:
I dont think we need to send a connected event as we already have media ready , Both media:ready and connected will trigger at the same time